frontend/src/pages/CatDetails/ui/CatDetails.tsx

```diff
@@ -76,7 +76,7 @@ const CatDetails = () => {
 						<div className={"askButtons"}>
 							<Button
 								width={"20vh"}
-								primary
+								variant={"primary"}
```

Comment:
non-blocking: можно без фигурных скобок
```suggestion
								variant="primary"
```